src/features/spam-hammer/index.js

```diff
@@ -4,27 +4,48 @@ const text = require('../../text')
 const { allowWhiteListChat } = require('../../middlewares/allowed-chat')
 const { adminRequiredSilenced } = require('../../middlewares/admin-required')
 const { Message } = require('../../models')
+const { installKeyboardActions, keyboardUnspamUser } = require('./keyboards')
 
 
-function handleEachMessage({ message, from, chat }, next) {
+async function handleEachMessage({
+  message, from, chat, getHammer, getChat, privateChannel,
+}, next) {
   debug(`handleEachMessage(messageId: ${message.message_id}, fromId: ${from.id}, chatId: ${chat.id}`)
+
   if (chat.type !== 'private') {
     Message.create({
       messageId: message.message_id,
       chatId: chat.id,
       authorId: from.id,
       date: message.date,
     }).catch((error) => {
-      debug('message create', error)
+      debug('handleEachMessage:Message.create ERROR', error)
     })
-  }
 
-  next()
+    next()
+
+    const hammer = getHammer()
```

Comment:
Should be more tested.

Remove it check if user kicked right after command `!spam`